ansible/vars/enikshay/unx40423.pub

```diff
@@ -0,0 +1 @@
+ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQDvNQLnUdNc+Y6gk+IcYjeasQ7hyCFn+OFBeNfe5hTZvAnOEPDzstTrs+SOcNLa/Qoc7XxfxIQBWKFkQUrJ17vKmm6suE6rXO6MmfAtAsxSQMPPTc6BjakraVncP0OcnG6qVV6P+r0HQJTicBMSU08heTsZfxBXIQbt39bjcjJiaA4E3wF57cgitcz0AEfmi9L5ah+0L2dST3FCHWqU9uXbdHE560PHW6PhXfisEx7F7XQahsniwOpT7dkmxaJKL3Pm7QTZdatUrJ5TceJxYisybxUKzmGDBuAONtIgcJ+dSRxiZZqlqu/rq8EwPpOYhndFafKm28u9EuhG16Y21Q/X unx40423@ubuntu
```

Comment:
we don't need this, i think

Reply:
@javierwilson this is the different pub key that should be added for ssh for new user `unx40423` like we had for msp?

Reply:
got it. yeah we keep those in files.